### PR TITLE
Not require registration checks on admin user creation

### PR DESCRIPTION
**Pull request by s-tyshchenko**
Issue: https://github.com/humhub-contrib/legal/issues/38

### CODE REVIEW

File: models/RegistrationChecks.php [hunk 1]
```diff
@@ -48,6 +48,10 @@ public function rules()
         $module = Yii::$app->getModule('legal');
         $rules = [];
 
+        if (Yii::$app->user->isAdmin()) {
```
Comment (luke-) [inline models/RegistrationChecks.php:51]: In this case admin never have to agree or? 
Alternatively maybe something like: Current User is not Registration User or a check based on Scenario?  

In any case we should add here some comments that it is for the case  admin use creation.

File: models/RegistrationChecks.php [hunk 2]
```diff
@@ -175,17 +179,17 @@ public function save()
         }
 
         if ($this->showTermsCheck()) {
-            $module->settings->user($this->user)->set(static::SETTING_KEY_TERMS, true);
+            $module->settings->user($this->user)->set(static::SETTING_KEY_TERMS, (bool) $this->termsCheck);
```
Comment (luke-) [inline models/RegistrationChecks.php:182]: We should only store it, when accepted.

**Review by luke- — APPROVED** [review]

**Review by luke- — CHANGES_REQUESTED** [review]
@s-tyshchenko Thanks, can you please see my comments? Please also increase the module version in the file `module.json`

File: models/RegistrationChecks.php (hunk 1 above)
Comment (s-tyshchenko) [reply models/RegistrationChecks.php:52]: In this case admin can leave registration checks unchecked and after the first login, the user will have to accept them himself.

Comment (luke-) [reply models/RegistrationChecks.php:52]: Yes, that would be ideal, the admin should not check it and the user should have to accept it himself.

**Review by luke- — CHANGES_REQUESTED** [review]

**Review by luke- — APPROVED** [review]

**Pull request merged**